tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -131,6 +131,47 @@ class BaseExampleGenExecutor(
   ExampleGen can override `GenerateExamplesByBeam`.
   """
 
+  def __init__(self):
+    super(BaseExampleGenExecutor, self).__init__()
+    self.input_signature = {}
+    self.output_signature = {}
```

Comment:
This shouldn't be required after changing how `beam_io_signature()` works; see my next comment.

Reply:
Done.